### PR TITLE
INFRA-284 Adding all missing features to the current working version 

### PR DESCRIPTION
**Pull request by Schife**
After this merge, this branch will replace broken master

### CODE REVIEW

**Comment by rnicoll** [comment]
@Schife Is this okay to merge? What's the deployment plan if it goes in? In particular I'd want to deploy on a non-critical Jenkins first, I'm thinking CI03?

File: build.gradle [hunk 1]
```diff
@@ -50,8 +52,10 @@ dependencies {
     implementation 'org.apache.commons:commons-csv:1.1'
     implementation group: 'org.jetbrains', name: 'annotations', version: '13.0'
     implementation 'com.microsoft.azure:azure:1.3.0'
+    implementation 'org.yaml:snakeyaml:1.23'
     testImplementation "org.junit.jupiter:junit-jupiter-api:5.5.2"
     testRuntimeOnly "org.junit.jupiter:junit-jupiter-engine:5.5.2"
+    testCompile group: 'org.assertj', name: 'assertj-core', version: '3.6.1'
```
Comment (chrisr3) [inline build.gradle:58]: `testCompile` is deprecated and should be replaced by `testImplementation`.

File: src/main/java/com/r3/testing/KubesReporting.java [hunk 2]
```diff
@@ -29,13 +29,15 @@
 import org.gradle.api.tasks.OutputDirectory;
 import org.gradle.api.tasks.TaskAction;
 import org.gradle.api.tasks.testing.Test;
+import org.gradle.internal.impldep.org.apache.commons.io.input.ReversedLinesFileReader;
 import org.gradle.internal.logging.ConsoleRenderer;
```
Comment (chrisr3) [inline src/main/java/com/r3/testing/KubesReporting.java:33]: Relying on Gradle's shaded version of _anything_ is likely to be brittle.

**Comment by Schife** [comment]
Now that the oracle changes have been merged I'll have to update this PR and then test against a version of Corda

**Comment by rnicoll** [comment]
@Schife If you can look at Chris' comments, and then let me know what the test plan for this is, I think this is otherwise good to go and would be good to get merged.

**Comment by Schife** [comment]
@rnicoll Sure. Would like to get this https://github.com/corda/gradle-distributed-testing-plugin/pull/39 PR merged in first and then I'll address the comments here. As for testing plan I will cut a snapshot release of this plugin and then raise a test PR on Corda to validate that we are still compatible. Once that is done we can merge this in

**Comment by rnicoll** [comment]
Okay merged #39 @Schife

**Comment by Schife** [comment]
Features we're added to version 1.3.0

**Pull request closed without merging**